### ISSUE TITLE
Versioned Gcc toolchains for copr

### ISSUE THREAD

**Issue by LecrisUT** [jwakely/pkg-gcc-latest#13]
It would be nice to provide versioned gcc toolchains on copr since Fedora does not have versioned packages of these.

**Comment by LecrisUT** [comment]
I mean some that provides `/usr/bin/gcc-11` or rather `/opt/gcc-11/bin/gcc-11`?

**Comment by jwakely** [comment]
No, sorry, this is meant to provide "the latest GCC snapshot" only, and just for testing and evaluation purposes.

I'm not interested in providing any kind of general purpose alternative GCC packages for Fedora, especially not multiple versions of them in parallel. That's out of scope for this project.
